src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: "Meet New Relic AI, your observability assistant"
+metaDescription: "New Relic AI is a GenAI assistant for observability that will help you understand and improve your system"
+tags:
+    - APM
+    - New Relic AI
+    - Grok
+---
+
+New Relic AI is designed to help engineers improve observability. We combined OpenAI's large language models (LLMs) with our unified telemetry data platform to create an AI assistant that deeply understands how both your system and New Relic function. NRAI helps you demystify the complexity of your stack, solve service outages and improve your efficiency simply by asking real, human questions.
+
+<Video
+  type="youtube"
+  id="XMSUVmNKVyM"
+/>
+
+## Work with our AI [#work-with-NRAI]
+
+Our AI uses natural language to get to the heart of your specific system. You can interact with NRAI in three ways: chat, the NRQL console, and CodeStream. 
+
+### Ask our AI [#ask-grok]
+
+Our AI has been trained to communicate with engineers using common questions and plain language. Let's say you run an online store and you start receiving alerts that response times for your `Checkout-service` are lagging. You don't have time to dig into the issue so you can just ask our AI "What's on fire?" or "What should I handle first today?" NRAI can tell you what's broken in your system in plain language, and help you visualize and explore data with charts and tables.. 
+
+You can work with our AI any time you want by using our interactive chat experience. All you have to do is click the **Ask AI** button available on the top right of most New Relic platform pages. 
+
+### NRQL console [#NRQL-console]
+
+When you chat with our AI it translates your questions into queries so you can visualize your data as a chart, graph, table, or billboard. [NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/) is a powerful tool that can be used to gain insights into your applications and infrastructure but when you're in a hurry you might not have time to find the right query to get the data you need. If you're struggling to get a query to work, ask our AI to help you. You can use plain speak to ask questions like, "How is this service performing?" or "How has this service been performing over the past 24 hours?"
+
+To enable NRAI in the NRQL console: 
+
+1. Go to the user menu at the bottom left of the New Relic UI..
+2.  Click the **Features** tab.
+3. Scroll down to the **New Relic AI** section and click **Enable**.
+
+### Use our AI in CodeStream [#codestream]
```

Comment:
Do you need to broaden this out to include errors inbox in the header?

Use our AI in CodeStream and errors inbox